### PR TITLE
Math magicians

### PR DESCRIPTION
**Pull request by AACHOURMOHAMED**
- [ ] Add ui to the calucaltor

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,12 @@
-import logo from './logo.svg';
 import './App.css';
+// eslint-disable-next-line import/no-named-as-default
+import Calculator from './components/Calculator';
 
-function App() {
+// eslint-disable-next-line arrow-body-style
+const App = () => {
   return (
```
Comment (TedLivist) [inline src/App.js]: - [ ] The App component should be class-based as well. Kindly do that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @AACHOURMOHAMED,

Good job so far!
✔️ App design matches wireframe
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly format your README to be well documentation using [the Microverse template](https://github.com/microverseinc/readme-template)

- [ ] You are required to have deleted unnecessary boilerplate files. Kindly delete the highlighted files in the images below

  src folder | public folder
  :---:|:---:
  src-1 | public-1

- [ ] **_Check the comments under the review._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — CHANGES_REQUESTED** [review]
Hi @AACHOURMOHAMED, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] As requested by the previous code reviewer, Kindly format your README to be well documented using [the Microverse template](https://github.com/microverseinc/readme-template) instead of having the README generated from `create react app`.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by AACHOURMOHAMED** [comment]
I changed the readme file

**Review by Lordkaito — APPROVED** [review]
Hi @AACHOURMOHAMED, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**